.github/workflows/publish-docs.yml

```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches:
         - master
-        - documentation/test
+        - release-*
```

Comment:
I continue to be of the opinion that we should generate versioned docs *at least* for tagged versions without an accompanying release-* branch. And preferably for all tagged versions which are new enough to have sphinx-based documentation.